src/app/app.component.ts

```diff
@@ -102,4 +102,6 @@ export class AppComponent {
   copy(item: any) {
     console.log('copy', item);
   }
+
+  visibility = false;
```

Comment:
Add check that the initial value is true, and see what happens.

Reply:
Thx for finding this. I added a check after global create is called and the tooltip will show up.

Reply:
Please add a test for this use case, and update the docs.

Reply:
Is it ok to add a second test suite and create a second component? Otherwise I would not know how to setup the pre-condition already in the component.

Reply:
Sure